lib/src/dotenv.dart

```diff
@@ -61,9 +61,19 @@ class DotEnv {
   /// Loads environment variables from the env file into a map
   /// Merge with any entries defined in [mergeWith]
   Future<void> load(
-      {String fileName = '.env',Parser parser = const Parser(),Map<String, String> mergeWith = const {}}) async {
+      {String fileName = '.env',Parser parser = const Parser(),Map<String, String> mergeWith = const {}, bool optional = false}) async {
```

Comment:
I'd use multiple lines to clarify the sign

Reply:
The whole file doesn't meet dart format. One simple `flutter format lib` would fix this, but I didn't want to let this PR get out of scope.